library/cmake/Modules/FindPHP.cmake

```diff
@@ -87,26 +101,29 @@ MARK_AS_ADVANCED(
 	PHP_VERSION_STRING
 )
 
+set(PHP_DEVELOPMENT_COMPONENT_ACCPETED TRUE)
+if(PHP_USE_DEVELOPMENT_COMPONENT)
+	FIND_PATH(zend_path
+		NAMES "zend.h"
+		PATHS ${PHP_INCLUDE_DIRS}
+	)
+	if(zend_path)
+		set(PHP_DEVELOPMENT_COMPONENT_FOUND TRUE)
```

Comment:
What is the difference between PHP_DEVELOPMENT_COMPONENT_FOUND and PHP_DEVELOPMENT_COMPONENT_ACCPETED? It looks like PHP_DEVELOPMENT_COMPONENT_FOUND is never used.

Reply:
`PHP_DEVELOPMENT_COMPONENT_ACCEPTED` is an unarguably horrible name for it's purposes.  It's meant for giving the a-okay to say `PHP_FOUND = TRUE`: if we're not looking for it, or we're looking for it and find it, it's set to true.  Willing to take suggestions for a much better name... XD